demo/index.html

```diff
@@ -21,24 +21,15 @@
         background: #eee;
         border: 1px solid #ddd;
         font-size: 16px;
-        margin: 8px;
+        margin: 20px;
         padding: 8px;
       }
     </style>
   </head>
 
   <body>
-    <div class="container">
-      <button data-tippy="Auto">Auto</button>
-
-      <h2>Group</h2>
-      <button class="group">Group</button>
-      <button class="group">Group</button>
-      <button class="group">Group</button>
-      <button class="group">Group</button>
-
-      <h2>Event delegation</h2>
-      <button class="delegation" data-tippy-content="Event delegation"></button>
-    </div>
+    <div class="container"></div>
+    <!-- prettier-ignore -->
+    <script> </script>
```

Comment:
?

Reply:
Fixes a Chrome transition bug that's existed for years. I'm using `showOnInit` in the demo and the tooltips transition weirdly on load without this hack. A script tag with an empty space fixes it (so weird, I know)

Reply:
https://stackoverflow.com/a/42969257/9470278

Reply:
Huh, interesting. Please, add a comment about this for the future generations 😉.